publication-rest/src/test/java/no/unit/nva/publication/update/UpdatePublicationHandlerTest.java

```diff
@@ -750,6 +751,25 @@ void shouldReturnForbiddenWhenContributorWithoutCristinIdUpdatesResource()
         assertThat(response.getStatusCode(), is(equalTo(HttpURLConnection.HTTP_UNAUTHORIZED)));
     }
 
+    @Test
+    void shouldNotReturnHiddenFilesWhenUserIsOwner() throws IOException, BadRequestException {
```

Comment:
This seems weird?

Reply:
(for now) only curator and editor should see hidden files. Maybe even less visibility in the future (next week). Hidden files is typically a license file or dublin core or something the curator puts there. Is the test weird or the requirement? 😅